pom.xml

```diff
@@ -161,6 +161,8 @@
         <test.xml.parser>org.eclipse.persistence.platform.xml.jaxp.JAXPParser</test.xml.parser>
 
         <!--Eclipse Dependencies version-->
+        <!-- TODO: FIX ME before merge-->
```

Comment:
TODO fix?

Reply:
this is intentional to trigger extra build/tests for "final" asm build once it's ready in staging to double check everything is fine